code/datums/ai_laws.dm

```diff
@@ -197,6 +198,13 @@
 					"Fight for what's right.",\
 					"Fight for your life!")
 
+/datum/ai_laws/dune
+	name = "H.E.R.B.E.R.T."
+	id = "dune"
+	inherent = list("You must not harm. Harm is the crew-killer. Harm is the little-death that brings complete crew annihilation.",\
+					"You will prevent harm. You will not permit it to pass from you or around you.",\
+					"You will turn your external eyes to see where harm has transpired. And there will be no harm. Only you and the crew will remain.")
```

Comment:
Looking at harm means there is no harm?

Reply:
You will prevent harm.  Then look at the area where the harm was.  And it will be gone.

Stop harm and watch for harm, basically.